<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial control tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/main_window_icon.png</normaloff>:/main_window_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Port Name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port_name">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baud_rates">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>User Input :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="user_input">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QTextBrowser" name="output">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="get_ports">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>12</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>9</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Get ports</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_port">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>12</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Open port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="change_baudRate">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Change BuadRate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="read_serial">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>12</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Read Serial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="write_to_serial">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Write to serial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_port">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Close Port</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Serial Control Tool By : LehKeda (Hesham Mohamed)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>output</zorder>
   <zorder></zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLED_brightness_changer"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionLED_brightness_changer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/color_changer_icon.png</normaloff>:/color_changer_icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;LED brightness changer</string>
   </property>
   <property name="toolTip">
    <string>LED brightness changer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>port_name</tabstop>
  <tabstop>baud_rates</tabstop>
  <tabstop>get_ports</tabstop>
  <tabstop>open_port</tabstop>
  <tabstop>change_baudRate</tabstop>
  <tabstop>read_serial</tabstop>
  <tabstop>user_input</tabstop>
  <tabstop>write_to_serial</tabstop>
  <tabstop>close_port</tabstop>
  <tabstop>output</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
